2016-01-13 09:48:56,693 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2016-01-13 09:48:56,702 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2016-01-13 09:49:34,099 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2016-01-13 09:49:34,110 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2016-01-13 09:49:34,114 INFO  [ckanext.report.report_registry] Report: BrokenLinkReportPlugin (BrokenLinkReportPlugin) broken-links
2016-01-13 09:50:15,834 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2016-01-13 09:50:15,846 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2016-01-13 09:50:55,903 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2016-01-13 09:50:55,914 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2016-01-13 09:50:55,967 ERROR [ckan.lib.mailer] SMTPRecipientsRefused({None: (501, '5.1.3 Bad recipient address syntax')},)
Traceback (most recent call last):
  File "/usr/lib/ckan/src/ckan/ckan/lib/mailer.py", line 82, in _mail_recipient
    smtp_connection.sendmail(mail_from, [recipient_email], msg.as_string())
  File "/usr/lib64/python2.6/smtplib.py", line 716, in sendmail
    raise SMTPRecipientsRefused(senderrs)
SMTPRecipientsRefused: {None: (501, '5.1.3 Bad recipient address syntax')}
2016-01-13 09:50:55,968 ERROR [root] Error: SMTPRecipientsRefused({None: (501, '5.1.3 Bad recipient address syntax')},); email: {'body': '2016-01-13 09:50:55.921743 Clean up stuck harvest jobs.\nHarvester is not running, or not frequently enough.\n', 'recipient_email': None, 'subject': 'harvest-job-cleanup Log', 'recipient_name': None}
2016-01-13 09:50:55.921743 Clean up stuck harvest jobs.
Harvester is not running, or not frequently enough.

Fetching tags from http://uat-catalog-fe-data.reisys.com/
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Fetching tags from http://uat-catalog-fe-data.reisys.com/
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
/usr/lib/ckan/lib/python2.6/site-packages/sqlalchemy/engine/reflection.py:47: SAWarning: Did not recognize type 'tsvector' of column 'anytext_tsvector'
  ret = fn(self, con, *args, **kw)
/usr/lib/ckan/lib/python2.6/site-packages/sqlalchemy/engine/reflection.py:47: SAWarning: Did not recognize type 'geometry(Geometry,4326)' of column 'wkb_geometry'
  ret = fn(self, con, *args, **kw)
Started gathering CKAN datasets identifiers: 2016-01-13 09:52:34.478835
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 1000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 2000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 3000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 4000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 5000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 6000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 7000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 8000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 9000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 10000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 11000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 12000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 13000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 14000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 15000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 16000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 17000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
Gathered 18000
Starting new HTTP connection (1): uat-catalog-fe-data.reisys.com
reading /etc/ckan/pycsw-all.cfg
Connecting to database
Running vacuum
reading /etc/ckan/pycsw-all.cfg
Connecting to database
Dropping FTS index
2016-01-13 09:54:56,655 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2016-01-13 09:54:56,665 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2016-01-13 09:54:56,672 INFO  [ckanext.harvest.logic.action.update] Harvest job run: {}

Traceback (most recent call last):
  File "/usr/bin/ckan", line 59, in <module>
    load_entry_point('PasteScript', 'console_scripts', 'paster')()
  File "/usr/lib/ckan/lib/python2.6/site-packages/paste/script/command.py", line 104, in run
    invoke(command, command_name, options, args[1:])
  File "/usr/lib/ckan/lib/python2.6/site-packages/paste/script/command.py", line 143, in invoke
    exit_code = runner.run(args)
  File "/usr/lib/ckan/lib/python2.6/site-packages/paste/script/command.py", line 238, in run
    result = self.command()
  File "/usr/lib/ckan/src/ckanext-harvest/ckanext/harvest/commands/harvester.py", line 114, in command
    self.run_harvester()
  File "/usr/lib/ckan/src/ckanext-harvest/ckanext/harvest/commands/harvester.py", line 270, in run_harvester
    jobs = get_action('harvest_jobs_run')(context, {})
  File "/usr/lib/ckan/src/ckan/ckan/logic/__init__.py", line 328, in wrapped
    result = _action(context, data_dict, **kw)
  File "/usr/lib/ckan/src/ckanext-harvest/ckanext/harvest/logic/action/update.py", line 621, in harvest_jobs_run
    source = harvest_source_show(context, {'id': job['source_id']})
  File "/usr/lib/ckan/src/ckan/ckan/logic/__init__.py", line 388, in wrapper
    return action(context, data_dict)
  File "/usr/lib/ckan/src/ckanext-harvest/ckanext/harvest/logic/action/get.py", line 36, in harvest_source_show
    source_dict = logic.get_action('package_show')(context, data_dict)
  File "/usr/lib/ckan/src/ckan/ckan/logic/__init__.py", line 328, in wrapped
    result = _action(context, data_dict, **kw)
  File "/usr/lib/ckan/src/ckan/ckan/logic/action/get.py", line 722, in package_show
    raise NotFound
ckan.logic.NotFound
